doc/contribute/bin_blobs.rst

```diff
@@ -72,12 +66,16 @@ corresponding :file:`module.yml` file. A specification for a blob must contain:
 - License text applicable to the blob or a reference to such text, in SPDX
   format
 
-The west extension used to fetch blobs will accept command line arguments that
-will allow the user to select the module or modules to fetch blobs from, as well
-as obtaining information and metadata on available blobs. This is required for
-user control and a uniform user experience regardless of the blob vendor.
+See the :ref:`corresponding section <modules-bin-blobs>` for a more formal
+definition of the fields.
+
+The :ref:`west extension command <west-blobs>` used to fetch blobs accepts
+command line arguments that allow the user to select the module or modules
+to fetch blobs from, as well as obtaining information and metadata on available
+blobs. This is required for user control and a uniform user experience
+regardless of the blob vendor.
 
-The west extension will only fetch and download the binary blobs themselves. Any
+The west extension only fetches and stores the binary blobs themselves. Any
```

Comment:
Maybe just:
```suggestion
 The west extension fetches and stores the binary blobs. Any 
```

it's given implicit with the following information that it doesn't do more than that

Reply:
I disagree here, I want to make it very clear that the tool only deals with the blobs. I've changed the following sentence a bit though.